language/move-core/types/src/fs.rs

```diff
@@ -0,0 +1,535 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::{de, export::Formatter, Deserialize, Deserializer, Serialize, Serializer};
```

Comment:
I would like to avoid adding this file to `move-core`. The reason is that it sits relatively low in libra's dependency tree, meaning that all crates in `libra` will basically depend on this crate. I would argue that moving this to `move-lang` seems to be a better location.

Reply:
The source language and the IR use the same spanned information. So maybe in the IR types, `move-ir/types`?